pytestqt/plugin.py

```diff
@@ -694,6 +696,25 @@ class _QtMessageCapture(object):
     def __init__(self, ignore_regexes):
         self._records = []
         self._ignore_regexes = ignore_regexes or []
+        self._previous_handler = None
+
+    def _start(self):
+        self._previous_handler = qInstallMsgHandler(self._handle)
+
+    def _stop(self):
+        qInstallMsgHandler(self._previous_handler)
+
+    @contextmanager
+    def disabled(self):
```

Comment:
Hmm perhaps... I named it from *"disabled within the context"*.

Either way, I think it is too late to rename it now without breaking semantic versioning... :no_mouth: